ffi/Cargo.toml

```diff
@@ -30,5 +30,5 @@ tracing-subscriber = { version = "0.3", features = ["std", "fmt", "local-time",
 
 [target.'cfg(windows)'.dependencies]
 symbol-rename-macro = { path = "./symbol-rename-macro" }
-winapi = "0.3" # FIXME: replace by windows / windows-sys crates
+winapi = { version =  "0.3", features = ["winerror"] } # FIXME: replace by windows / windows-sys crates
```

Comment:
question: Is it a leftover, or something necessary?

Reply:
we need `winapi::shared::winerror::ERROR_SUCCESS` for _tsssp_ but `winerror` module is behind the _winerror_ feature